### PR TITLE
[Benchmark] Fix amp level bug in some gpt tests

### PR DESCRIPTION
**Pull request by zhangbo9674**
### PR types

Bug fixes
### PR changes

Others
### Description

Fix amp level bug in some gpt tests

### CODE REVIEW

File: legacy/model_zoo/gpt-3/benchmarks/test_tipc/gpt/static/new_exec_pp_pir/benchmark_common/run_benchmark.sh [hunk 1]
```diff
@@ -35,6 +35,18 @@ function _set_params(){
     sharding_degree=${10:-"1"}      # (可选)
     sharding_stage=${11:-"1"}       # (可选)sharding case
     level=${12:-"o1"}               # o1|o2|o3
+
+    if [ $FLAGS_enable_pir_api == 1 ]; then
+        if [ ${level} == "o3" ]; then
+            ${level}="o2"
```
Comment (fightfat) [inline legacy/model_zoo/gpt-3/benchmarks/test_tipc/gpt/static/new_exec_pp_pir/benchmark_common/run_benchmark.sh]: 这里应该是 level="o2"吧？

File: legacy/model_zoo/gpt-3/benchmarks/test_tipc/gpt/static/new_exec_pp/benchmark_common/run_benchmark.sh [hunk 2]
```diff
@@ -35,6 +35,18 @@ function _set_params(){
     sharding_degree=${10:-"1"}      # (可选)
     sharding_stage=${11:-"1"}       # (可选)sharding case
     level=${12:-"o1"}               # o1|o2|o3
+    
+    if [ $FLAGS_enable_pir_api == 1 ]; then
```
Comment (fightfat) [inline legacy/model_zoo/gpt-3/benchmarks/test_tipc/gpt/static/new_exec_pp/benchmark_common/run_benchmark.sh]: 这里改成这样吧，    if [[ $FLAGS_enable_pir_api == "1" || $FLAGS_enable_pir_api == "True" ]]; then

File: legacy/model_zoo/gpt-3/benchmarks/test_tipc/gpt/static/new_exec_pp_pir/benchmark_common/run_benchmark.sh [hunk 3]
```diff
@@ -35,6 +35,18 @@ function _set_params(){
     sharding_degree=${10:-"1"}      # (可选)
     sharding_stage=${11:-"1"}       # (可选)sharding case
     level=${12:-"o1"}               # o1|o2|o3
+
+    if [ $FLAGS_enable_pir_api == 1 ]; then
```
Comment (fightfat) [inline legacy/model_zoo/gpt-3/benchmarks/test_tipc/gpt/static/new_exec_pp_pir/benchmark_common/run_benchmark.sh]: 这里改成这样吧，    if [[ $FLAGS_enable_pir_api == "1" || $FLAGS_enable_pir_api == "True" ]]; then

File: legacy/model_zoo/gpt-3/benchmarks/test_tipc/gpt/static/new_exec_pp/benchmark_common/run_benchmark.sh (hunk 2 above)
Comment (zhangbo9674) [reply legacy/model_zoo/gpt-3/benchmarks/test_tipc/gpt/static/new_exec_pp/benchmark_common/run_benchmark.sh]: done

File: legacy/model_zoo/gpt-3/benchmarks/test_tipc/gpt/static/new_exec_pp_pir/benchmark_common/run_benchmark.sh (hunk 3 above)
Comment (zhangbo9674) [reply legacy/model_zoo/gpt-3/benchmarks/test_tipc/gpt/static/new_exec_pp_pir/benchmark_common/run_benchmark.sh]: done

**Review by zhiqiu — APPROVED** [review]
LGTM

**Review by ZHUI — APPROVED** [review]

**Pull request merged**